google-datacatalog-sisense-connector/docs/developer-resources/sisense-apis.md

```diff
@@ -0,0 +1,10 @@
+# Sisense APIs
+
+The below API docs were used as references to develop this connector:
+
+1. [Sisense REST API v1.0 Reference (Sisense V6 and
+   higher)](https://sisense.dev/reference/rest/v1.html)
```

Comment:
Any reason you are using the v1.0 and not v2.0 (latest) of their REST API? 
https://sisense.dev/reference/rest/v2/?platform=linux&spec=L2021.3.0

Reply:
This is the version available in the Sandbox environment: `Sisense Windows Version: 8.2.5.11026 v1`.